src/ansi.rs

```diff
@@ -39,14 +39,14 @@ const SYNC_UPDATE_TIMEOUT: Duration = Duration::from_millis(150);
 /// Maximum number of bytes read in one synchronized update (2MiB).
 const SYNC_BUFFER_SIZE: usize = 0x20_0000;
 
-/// Number of bytes in the synchronized update DCS sequence before the passthrough parameters.
-const SYNC_ESCAPE_START_LEN: usize = 5;
+/// Number of bytes in the BSU/ESU CSI sequences.
+const SYNC_ESCAPE_LEN: usize = 8;
```

Comment:
I'm not sure what you mean? This way we have a constant to use in the size of the byte arrays, otherwise we'd have no way to guarantee both arrays stay the same length, which is important.

Reply:
There's two arrays though. I can't define a single constant based on the length of two arrays. But I can define two arrays using a single length constant.

They **must** have the same length, so we should use a constant to synchronize them.